module/zfs/zio.c

```diff
@@ -513,7 +513,7 @@ zio_decrypt(zio_t *zio, abd_t *data, uint64_t size)
 	/*
 	 * If this is an authenticated block, just check the MAC. It would be
 	 * nice to separate this out into its own flag, but for the moment
```

Comment:
This is something that should probably be fixed, although aside from fixing the comment to stop saying that we are out of bits, I do not think there is anything else to do here. I am currently at SFO airport waiting for my flight to begin boarding. I will revise this and repush when I am back home.

Reply:
I might also clean this up in another PR, but that is not likely to be done immediately. My schedule for the next week is likely not only full, but slightly overfilled at this point.

Reply:
Good find, yes this is something we can safely leave as future work.

Reply:
I have fixed the comment.